views/edit_group.erb

```diff
@@ -0,0 +1,11 @@
+
+<form method="POST" action="/new_group">
```

Comment:
This will POST to /new_group, most likely this is not the desired behaviour since this will create a new group and not update an existing one.